docs/getting_started_rules.md

```diff
@@ -0,0 +1,152 @@
+
+These rules don't apply to custom rules sets, you're free to build your own rules in the way they suit you best.
+
+### Rule Matches
+A rule returns it findings by returning a list of `RuleMatch` object. A Match is an object containing information about the finding:
+
+* Path to the resource. Used for returning line/column information of the finding in the file.
+* The message about the finding. This is a specific error message that can be formatted to contain detailed information about the error (It's not the Rule's description)
+
+## Creating a new rule
+The following steps describe the basics on how to create a new rule.
+
+*As an example we use `MyNewRule` as the rule to be added.*
+
+### Select a new Rule ID
+Go to the [rules documentation](./rules.md) and find an appropriate available Rule ID, based on the correct [category](./rules.md#categories).
+
+### Create new Rule
+Based on the use case and [rule category](./rules.md#categories), create a new Python file in the correct namespace of the [rules folder](/src/cfnlint/rules). Use the name of the Rule, so the filename of `MyNewRule` is `mynewrule.py`.
+
+Use the following skeleton code as a starting point of your new rule:
+
+```python
+"""
+  Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this
+  software and associated documentation files (the "Software"), to deal in the Software
+  without restriction, including without limitation the rights to use, copy, modify,
+  merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+  permit persons to whom the Software is furnished to do so.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+  INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+  PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+  SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+"""
+from cfnlint import CloudFormationLintRule
+from cfnlint import RuleMatch
+
+
+class MyNewRule(CloudFormationLintRule):
+    """Rule description """
+    id = '' # New Rule ID
+    shortdesc = '' # A short description about the rule
+    description = '' # (Longer) description about the rule
+    source_url = '' # A url to the source of the rule, e.g. documentation, AWS Blog posts etc
+    tags = [] # A set of tags (strings) for searching
+
+    def match(self, cfn):
+        """Basic Rule Matching"""
+
+        matches = list()
+
+        # Your Rule code goes here
+
+        return matches
+```
+Fill in the Meta data (`id`, `shortdesc`, `description`, `source_url`, `tags`) of your new rule, there are required attributed needed for processing the rule.
```

Comment:
`there are required attributed needed for processing the rule.` -> `these are required attributes needed for processing the rule`